test/async-hooks/verify-graph.js

```diff
@@ -108,7 +108,8 @@ module.exports = function verifyGraph(hooks, graph) {
 
   for (const type in expTypes) {
     assert.strictEqual(typeSeen[type], expTypes[type],
-                       `Expecting type '${type}' in graph`);
+                       `Type '${type}': expecting: ${expTypes[type]} found: ${
+                         typeSeen[type]}`);
```

Comment:
Micro-nit that you can ignore: Not a fan of the "hide the whitespace in ${}" trick as it seems less readable to me than simply splitting up the template string into two template strings. Would rather see multi-line string concatenation.

Reply:
👍